Jenkinsfile

```diff
@@ -51,27 +51,57 @@ try {
             node('docker') {
                 deleteDir()
                 git 'https://github.com/jenkins-infra/azure.git'
+                /* Create an empty terraform variables file so that everything can
+                 * be overridden in the environment
+                 */
                 sh "echo '{\"prefix\":\"${tfPrefix}\"}' > ${tfVarFile}"
+
+                /*
+                 * Three following actions must be remove once terraform is greater than 0.8
+                 * Cfr INFRA-1288
```

Comment:
cfr is equivalent to cf (https://en.wikipedia.org/wiki/Cf.)
And seriously, in terraform 0.9, `terraform init` expect to be run at the root of the project where everything is located (plans,variables,...), and it also need a backend configuration file if we use another backend than local.
But for an unkowned reason, running Terraform 'operations' on subdirectories (like terraform apply plans) also complains for this missing backend configuration file.
This is why I define it in two places (root && plans), another solution would have been to reorganise the project structure (move all plans at the root) but in terraform 0.10 it should be possible to init subdirectories.